artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PageTimedWriter.java

```diff
@@ -118,18 +120,20 @@ public void incrementTask() {
       pendingTasksUpdater.incrementAndGet(this);
    }
 
-   public void addTask(OperationContext context,
+   public int addTask(OperationContext context,
                                     PagedMessage message,
                                     Transaction tx,
-                                    RouteContextList listCtx) {
+                                    RouteContextList listCtx, boolean useFlowControl) {
 
       logger.trace("Adding paged message {} to paged writer", message);
 
+      // the module using the page operation should later call flowControl from this class.
```

Comment:
The module using the page operation should later call `writeFlowControl`, right?